install_config/configuring_azure.adoc

```diff
@@ -11,26 +11,45 @@
 toc::[]
 
 == Overview
-{product-title} can be configured to access an
-link:https://azure.microsoft.com/en-us/services/storage/disks/[Azure
+{product-title} can be configured to access
+link:https://azure.microsoft.com/en-us/services/storage/disks/[Microsoft Azure
 infrastructure], including
 xref:../install_config/persistent_storage/persistent_storage_azure.adoc#install-config-persistent-storage-persistent-storage-azure[using
-Azure disk as persistent storage] for application data. After Azure is
+Azure disk as persistent storage] for application data. After Microsoft Azure is
 configured properly, some additional configurations need to be completed on the
 {product-title} hosts.
 
 [[configuring-azure-permissions]]
 == Permissions
-Configuring Azure for {product-title} requires the following role:
+Configuring Microsoft Azure for {product-title} requires the following role:
 
 [cols='1,3']
 |===
 
 | Contributor
-| To create and manage all types of Azure resources.
+| To create and manage all types of Microsoft Azure resources.
 
 |===
 
+[[configuring-azure-prerequisites]]
+== Prerequisites
+
+* If you are using Microsoft Azure Disk as an OpenShift Persistent Volume for  {product-title}'s latest version (or lower upto version 3.5), you must enable Azure Cloud Provider.
+* All virtual machines (VM) configured as OpenShift nodes, must belong to a single
+resource group.
+* Microsoft Azure VM names:
+  ** Must be same as OpenShift node names, and
+  ** These names must not be included any capital letters.
+* If you plan to use  Azure Managed Disks:
+  ** {product-title} version 3.7 or later is required.
+  ** You must create VMs with Azure Managed Disks.
+* If you plan to use unmanaged disks:
+  ** You must create VMs with unmanaged disks.
```

Comment:
Could this be one line with the above?

Reply:
Yes, I just used it to format it similar to the point above:
![image](https://user-images.githubusercontent.com/23069445/43114089-9092dd36-8f40-11e8-9f15-f1bfb791ef8e.png)

If this is not ideal I can change it to a single line.